services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy

```diff
@@ -162,4 +162,53 @@ class BandwidthSpec extends BaseSpecification {
         cleanup: "Delete created flow"
         flow?.id && northboundService.deleteFlow(flow.id)
     }
+
+    def "Able to update bandwidth to maximum link speed without using alternate links"() {
+        given: "Two active neighboring switches"
+        def switches = topology.getActiveSwitches()
+        def links = northboundService.getAllLinks()
+        def (Switch srcSwitch, Switch dstSwitch) = [switches, switches].combinations()
```

Comment:
You can just pick a link and use its src/dst switch.
```
def isl = topology.getIslsOverActiveSwitches().first()
def srcSwitch = isl.srcSwitch
def dstSwitch = isl.dstSwitch
```

Reply:
ok, thanks.